runtime/makelib/targets.mk.linux.inc.ftl

```diff
@@ -418,7 +418,7 @@ endif
 
 <#if uma.spec.processor.arm>
 $(patsubst %.s,%.o,$(filter %.s,$(UMA_FILES_TO_PREPROCESS))): %$(UMA_DOT_O): %.s
-	sed -f $(UMA_PATH_TO_ROOT)tr.source/codegen/makefiles/armasm2gas.sed $*.s > $*.S
+	sed -f $(UMA_PATH_TO_ROOT)compiler/codegen/makefiles/armasm2gas.sed $*.s > $*.S
```

Comment:
The path should be `compiler/trj9/build/scripts/armasm2gas.sed`

Reply:
Fixed in 9ba3ffe